crates/api_common/src/context.rs

```diff
@@ -31,6 +35,9 @@ impl LemmyContext {
   pub fn pool(&self) -> &DbPool {
     &self.pool
   }
+  pub async fn conn(&self) -> LemmyResult<DbPooledConn> {
```

Comment:
Is LemmyContext.pool() still being used?

Reply:
Yes